certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+        "summary": "Delete envelope by id",
+        "description": "Delets an envelope identified by its id",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        },
+        "x-ms-visibility": "advanced"
+      },
+      "put": {
+        "operationId": "PutEnvelopesEnvelopeId",
+        "summary": "Update envelope settings",
+        "description": "Updates Settings of a given envelope.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/EnvelopeSettings"
+            },
+            "required": true,
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/publish-status": {
+      "put": {
+        "operationId": "PutEnvelopesEnvelopeIdPublishStatus",
+        "summary": "Update publish status for envelope",
+        "description": "Updates the publish status of a given envelope",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "published": {
+                  "type": "boolean",
+                  "title": "Published",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "x-ms-visibility": "important"
+                }
+              },
+              "example": {
+                "published": true
+              },
+              "x-ms-visibility": "important",
+              "required": [
+                "published"
+              ]
+            },
+            "required": true,
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
```

Comment:
As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/publishing-an-envelope